### ISSUE TITLE
[devops] Prepare Merge bezos-main to staging

### ISSUE THREAD

**Issue by ladytrekker** [KlimaDAO/klimadao#1020]
## Description

The main branch "bezos-main" is regularly rebased with the main "staging" branch from the Klimadao repo.

This is because at some point the branch "bezos-main" will be merged into "staging".
Therefore, it should be aligned with "staging" all the time and also be ahead of "staging" in the commit history in order to prevent major merge conflicts.

Before the final merge can happen, some todos have to be done before.

## TODO
- [ ] add carbonmark to translation.io
- [x] make sure that all github actions are aware of the new repo "carbonmark" (sync with translation.io e.g.)
- [x] Vercel main production server should be aware of the new repo "carbonmark"
- [x] Go over ReadMe again, make sure carbonmark is added everywhere
- [x] Check if all envs are set (locally, on Vercel, mentioned in env.example)
